PatientWebApplication/UserMicroserviceApi/Controllers/PatientController.cs

```diff
@@ -67,11 +61,25 @@ public IActionResult SaveImg([FromForm] FileModel file)
 
         }
 
+        [HttpGet("getimage/{fileName}")]
+        [Authorize(Roles = "patient")]
+        public IActionResult GetImage(string fileName)
+        {
+            string path = _env.WebRootPath;
```

Comment:
Variable 'path' directly forward in method GetImage(path, fileName) and after remove variable 'path'